commands/issue/create/issue_create.go

```diff
@@ -184,7 +182,7 @@ func NewCmdCreate(f *cmdutils.Factory) *cobra.Command {
 	issueCreateCmd.Flags().StringVarP(&opts.Title, "title", "t", "", "Supply a title for issue")
 	issueCreateCmd.Flags().StringVarP(&opts.Description, "description", "d", "", "Supply a description for issue")
 	issueCreateCmd.Flags().StringVarP(&opts.Labels, "label", "l", "", "Add label by name. Multiple labels should be comma separated")
-	issueCreateCmd.Flags().StringVarP(&opts.Assignees, "assignee", "a", "", "Assign issue to people by their ID. Multiple values should be comma separated ")
+	issueCreateCmd.Flags().StringSliceVarP(&opts.Assignees, "assignee", "a", []string{}, "Assign issue to people by their ID. Multiple values should be comma separated ")
```

Comment:
The description should change. Since you're using `StringSliceVarP` (which is appropriate), together with  comma-separated values, the user can use `-a` multiple times. Eg. `glab issue create -t "Some title" -a "profclems,anotherUser"  -a zemzale`

```suggestion
	issueCreateCmd.Flags().StringSliceVarP(&opts.Assignees, "assignee", "a", []string{}, "Assign people by their `usernames`")
```

Reply:
Yeah makes total sense. I will fix this up probably next week.